packages/maskbook/src/plugins/Trader/contracts/uniswap/useQuoterContract.ts

```diff
@@ -5,5 +5,5 @@ import type { AbiItem } from 'web3-utils'
 
 export function useQuoterContract() {
```

Comment:
@guanbinrui suggest, abstract a contract hook factory method

Reply:
Let's keep it stupid and simple (KISS) first. 😄